### ISSUE TITLE
Buffalo Generate Template Naming does not work for ressource name with two words

### ISSUE THREAD

**Issue by kteb** [gobuffalo/buffalo#1860]
I wasn't sure it was linked to #1801 so I preferred to create a new issue, but basically the template folder doesn't have the right name. 

```
buffalo new coke
cd coke
buffalo generate resource NewResource name
buffalo pop create
buffalo pop migrate
buffalo dev
```
Going on http://localhost:3000/specification_types/ I receive the following error:

> new_resources/new.html: could not find template new_resources/new.html

Looking at the structure folder under template I found a folder new with inside an other folder resources instead of a folder new_resources that buffalo is expecting. If i copy all the folder structure in a folder templates/new_resources everything is working fine (with some fine tuning the includes). 

The fun fact is than the `buffalo destroy resource` is looking to destroy templates/new_resources

I'm using buffalo 0.15.3 on a Windows WSL system. 

I think the easiest way is to change this line https://github.com/gobuffalo/buffalo/blob/aa202b19ce380c61533b280daf56bf58de2f9a1b/genny/resource/resource.go#L51 with 

`g.Transformer(genny.Replace("folder-name", x))`

or change everywhere to have the resources looking for the right template folder. 

I wanted the feedback before creating a very simple or very complicated PR.

**Comment by mclark4386** [comment]
Yeah I think it's just that Folder isn't exactly what we want there (https://github.com/gobuffalo/flect/blob/master/name/folder.go#L20) I think https://github.com/gobuffalo/buffalo/blob/aa202b19ce380c61533b280daf56bf58de2f9a1b/genny/resource/resource.go#L48 should probably be changed to `Resource()` instead of `Folder()`, but I could be wrong...

**Comment by markbates** [comment]
Perhaps some tests will provide you the enlightenment you seek. 😉
